pkg/awsutils/vpc_ip_resource_limit.go

```diff
@@ -16,315 +16,323 @@ package awsutils
-	"x1e.32xlarge": 8,
-	"z1d.large":    3,
-	"z1d.xlarge":   4,
-	"z1d.2xlarge":  4,
-	"z1d.3xlarge":  8,
-	"z1d.6xlarge":  8,
-	"z1d.12xlarge": 15,
+	"c1.medium":     2,
+	"c1.xlarge":     4,
+	"c3.large":      3,
+	"c3.xlarge":     4,
+	"c3.2xlarge":    4,
+	"c3.4xlarge":    8,
+	"c3.8xlarge":    8,
+	"c4.large":      3,
+	"c4.xlarge":     4,
+	"c4.2xlarge":    4,
+	"c4.4xlarge":    8,
+	"c4.8xlarge":    8,
+	"c5.large":      3,
+	"c5.xlarge":     4,
+	"c5.2xlarge":    4,
+	"c5.4xlarge":    8,
+	"c5.9xlarge":    8,
+	"c5.18xlarge":   15,
+	"c5d.large":     3,
+	"c5d.xlarge":    4,
+	"c5d.2xlarge":   4,
+	"c5d.4xlarge":   8,
+	"c5d.9xlarge":   8,
+	"c5d.18xlarge":  15,
+	"cc2.8xlarge":   8,
+	"cr1.8xlarge":   8,
+	"d2.xlarge":     4,
+	"d2.2xlarge":    4,
+	"d2.4xlarge":    8,
+	"d2.8xlarge":    8,
+	"f1.2xlarge":    4,
+	"f1.4xlarge":    8,
+	"f1.16xlarge":   8,
+	"g2.2xlarge":    4,
+	"g2.8xlarge":    8,
+	"g3s.xlarge":    4,
+	"g3.4xlarge":    8,
+	"g3.8xlarge":    8,
+	"g3.16xlarge":   15,
+	"h1.2xlarge":    4,
+	"h1.4xlarge":    8,
+	"h1.8xlarge":    8,
+	"h1.16xlarge":   15,
+	"hs1.8xlarge":   8,
+	"i2.xlarge":     4,
+	"i2.2xlarge":    4,
+	"i2.4xlarge":    8,
+	"i2.8xlarge":    8,
+	"i3.large":      3,
+	"i3.xlarge":     4,
+	"i3.2xlarge":    4,
+	"i3.4xlarge":    8,
+	"i3.8xlarge":    8,
+	"i3.16xlarge":   15,
+	"i3.metal":      15,
+	"m1.small":      2,
+	"m1.medium":     2,
+	"m1.large":      3,
+	"m1.xlarge":     4,
+	"m2.xlarge":     4,
+	"m2.2xlarge":    4,
+	"m2.4xlarge":    8,
+	"m3.medium":     2,
+	"m3.large":      3,
+	"m3.xlarge":     4,
+	"m3.2xlarge":    4,
+	"m4.large":      2,
+	"m4.xlarge":     4,
+	"m4.2xlarge":    4,
+	"m4.4xlarge":    8,
+	"m4.10xlarge":   8,
+	"m4.16xlarge":   8,
+	"m5.large":      3,
+	"m5.xlarge":     4,
+	"m5.2xlarge":    4,
+	"m5.4xlarge":    8,
+	"m5.12xlarge":   8,
+	"m5.24xlarge":   15,
+	"m5a.large":     3,
+	"m5a.xlarge":    4,
+	"m5a.2xlarge":   4,
+	"m5a.4xlarge":   8,
+	"m5a.12xlarge":  8,
+	"m5a.24xlarge":  15,
+	"m5d.large":     3,
+	"m5d.xlarge":    4,
+	"m5d.2xlarge":   4,
+	"m5d.4xlarge":   8,
+	"m5d.12xlarge":  8,
+	"m5d.24xlarge":  15,
+	"p2.xlarge":     4,
+	"p2.8xlarge":    8,
+	"p2.16xlarge":   8,
+	"p3.2xlarge":    4,
+	"p3.8xlarge":    8,
+	"p3.16xlarge":   8,
+	"r3.large":      3,
+	"r3.xlarge":     4,
+	"r3.2xlarge":    4,
+	"r3.4xlarge":    8,
+	"r3.8xlarge":    8,
+	"r4.large":      3,
+	"r4.xlarge":     4,
+	"r4.2xlarge":    4,
+	"r4.4xlarge":    8,
+	"r4.8xlarge":    8,
+	"r4.16xlarge":   15,
+	"r5.large":      3,
+	"r5.xlarge":     4,
+	"r5.2xlarge":    4,
+	"r5.4xlarge":    8,
+	"r5.12xlarge":   8,
+	"r5.24xlarge":   15,
+	"r5a.large":     3,
+	"r5a.xlarge":    4,
+	"r5a.2xlarge":   4,
+	"r5a.4xlarge":   8,
+	"r5a.12xlarge":  8,
+	"r5a.24xlarge":  15,
+	"r5d.large":     3,
+	"r5d.xlarge":    4,
+	"r5d.2xlarge":   4,
+	"r5d.4xlarge":   8,
+	"r5d.12xlarge":  8,
+	"r5d.24xlarge":  15,
+	"t1.micro":      2,
+	"t2.nano":       2,
+	"t2.micro":      2,
+	"t2.small":      3,
```

Comment:
Oh, we had these wrong for t2 and t3.small. (Thanks for the comment diff to make it readable.)

Reply:
They got updated